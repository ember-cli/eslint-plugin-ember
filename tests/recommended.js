'use strict';

const rules = require('../lib/index.js').rules;

test('recommended rules', () => {
  const errors = [];

  Object.keys(rules).forEach((name) => {
    if (rules[name].meta.docs.recommended) {
      errors.push(name);
    }
  });

  expect(errors).toMatchSnapshot();
});
