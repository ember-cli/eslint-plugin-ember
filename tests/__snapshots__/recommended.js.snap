// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`recommended rules 1`] = `
Array [
  "alias-model-in-controller",
  "avoid-leaking-state-in-components",
  "closure-actions",
  "jquery-ember-run",
  "named-functions-in-promises",
  "no-attrs-snapshot",
  "no-capital-letters-in-routes",
  "no-empty-attrs",
  "no-function-prototype-extensions",
  "no-observers",
  "no-on-calls-in-components",
  "no-side-effects",
  "order-in-components",
  "order-in-controllers",
  "order-in-models",
  "order-in-routes",
  "routes-segments-snake-case",
  "use-brace-expansion",
  "use-ember-get-and-set",
]
`;
